stylesheets/vendor/bootstrap/partials/_forms.scss

```diff
@@ -22,6 +22,7 @@ legend {
   display: block;
   padding: 0;
   font-size: ($font-size-base * 1.5);
+  font-weight: 700;
   line-height: inherit;
   color: $legend-color;
   border: 0;
```

Comment:
I think you want `border: none` here instead of `border: 0`. In some browsers they're the same thing, in others, only `border: none` will work.

Reply:
Per https://stackoverflow.com/a/2922923/67911 - nevermind.

Reply:
i like that better, i think that was a remnant from bootstrap. changed